packages/smart-wallet/src/types.d.ts

```diff
@@ -27,7 +28,7 @@ export type InvitationMakers = Record<
   (...args: any[]) => Promise<Invitation>
 >;
 
-export type PublicSubscribers = Record<string, ERef<StoredFacet>>;
+export type PublicSubscribers = Record<string, PublicTopic>;
```

Comment:
Maybe?
```suggestion
export type PublicSubscribers = Record<string, PublicTopic | ResolvedPublicTopic>;
```

Reply:
`PublicTopic` is already a superset of `ResolvedPublicTopic` because `storagePath: ERef<string>` can be `storagePath: string`